internal/backends/qt/qt_window.rs

```diff
@@ -1943,14 +1944,15 @@ impl WindowAdapter for QtWindow {
         let maximized: bool = properties.is_maximized();
 
         cpp! {unsafe [widget_ptr as "QWidget*",  title as "QString", size as "QSize", background as "QBrush", no_frame as "bool", always_on_top as "bool",
-                      fullscreen as "bool", minimized as "bool", maximized as "bool"] {
+                      fullscreen as "bool", minimized as "bool", maximized as "bool", skip_taskbar as "bool"] {
 
             if (size != widget_ptr->size()) {
                 widget_ptr->resize(size.expandedTo({1, 1}));
             }
 
             widget_ptr->setWindowFlag(Qt::FramelessWindowHint, no_frame);
             widget_ptr->setWindowFlag(Qt::WindowStaysOnTopHint, always_on_top);
+            widget_ptr->setWindowFlag(Qt::Tool, skip_taskbar);
```

Comment:
Have you tried this out? For me the effect on macOS seems... less than desirable.

Reply:
For me setting `Qt::Tool` to true on a window makes it stay on top of _all other windows_ in the application. So with a multi-window application that seems not what we want. (this is on macOS)

Reply:
i'm not sure about that. Maybe Qt::Dialog?